bokeh/server/tornado.py

```diff
@@ -188,14 +190,30 @@ def __init__(self, applications, prefix, hosts,
 
             all_patterns.extend(app_patterns)
 
+            # add a per-app static path if requested by any app handler
+            # (there should be at most one that requests a static path)
```

Comment:
It's hard-coded to look at BokehJS, and there is nothing about per-app static files that a regular `StaticFileHandler` isn't suited for, I'm inclined to just change that comment.